packages/governance/src/types.js

```diff
@@ -321,9 +322,9 @@
  */
 
 /**
+ * @template P path for a paramManagerRetriever
```

Comment:
you can avoid edits to all the existing uses of `ParamChangeIssue` by giving the template a default value.
```suggestion
 * @template [P=unknown] path for a paramManagerRetriever
```